src/Packagist/WebBundle/Resources/views/Package/suggesters.html.twig

```diff
@@ -0,0 +1,29 @@
+{% extends "PackagistWebBundle::layout.html.twig" %}
+
+{% set showSearchDesc = 'hide' %}
+
+{% block head_additions %}<meta name="robots" content="noindex, nofollow">{% endblock %}
+
+{% block title %}Suggesters Packages - {{ name }} - {{ parent() }}{% endblock %}
+
+{% block content %}
+    <div class="row">
+        <div class="col-xs-12 package">
+            <div class="package-header">
+                <h2 class="title">
+                    <a href="{{ path("view_package", {name: name}) }}">{{ name }}</a> suggesters
+                    <small>({{ count }})</small>
+                </h2>
+            </div>
+        </div>
+    </div>
+
+    <section class="row">
+        <section class="col-md-12">
```

Comment:
BTW @stof, I agree for `section` case but not for `row` and `col-md-12`.

There is no parent `row` on the whole page. If we want to refactor, we'll have to refactor the hole templates folder.

Another great things we can do later is to use [bootlint](https://github.com/twbs/bootlint) to check if our templates are good according to Bootstrap conventions.